<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostScannerWindow</class>
 <widget class="QMainWindow" name="HostScannerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="networkLineEdit">
      <property name="placeholderText">
       <string>Enter Network (e.g., 192.168.1.0)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="subnetMaskLineEdit">
      <property name="placeholderText">
       <string>Enter Subnet Mask (e.g., 255.255.255.0)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar"/>
    </item>
    <item>
     <widget class="QTextEdit" name="resultTextEdit"/>
    </item>
    <item>
     <widget class="QTextEdit" name="aliveHostsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
